modules/test-utils/src/browser-automation/browser-test-driver.ts

```diff
@@ -80,6 +83,15 @@ export default class BrowserTestDriver extends BrowserDriver {
       }
 
       const result = await this._openPage(url, config);
+
+      if (config.coverage) {
+        const coverageResult = await this.page?.coverage.stopJSCoverage();
```

Comment:
Nit: It seems to me that this spreads out the coverage integration across base and subclass... For better encapsulation and symmetry, I feel that startJSCoverage and stopJSCoverage methods should both be called by either the BrowserDriver base class or this class.